src/dataflow/core/lispress.py

```diff
@@ -470,7 +460,9 @@ def unnest_line(
                 expr, idx = mk_value_op(value=int(n), schema="Long", idx=idx)
                 return [expr], idx, idx, var_id_bindings
             else:
-
+                if s in var_id_bindings:
```

Comment:
Bug fix: we could only handle let bindings that looked like indices (line 502 below).